### PR TITLE
[Doc Improvement][RSC][NAPI] - Updated Basic RSC perms to include TeamsActivity.Send.User

### PR DESCRIPTION
**Pull request by MSFTRickyCastaneda**
Updated table at line 53 to reflect the addition of a new Basic RSC permissions "TeamsActivity.Send.User" in addition to TeamsActivity.Send.Group

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7b02aa119183ba438707ffb1ebdc355227802d00/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 94 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/04516a44-4673-47b4-b460-f8cb3b6f782e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/graph-api/rsc/resource-specific-consent.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ A limited set of RSC permissions have been reviewed by Microsoft privacy and sec
 | Permission name | Action | Type: Delegated | Type: Application |
 | ----- | ----- | :-----: | :-----: |
 |`TeamsActivity.Send.Group`|Send activity feed notifications to users in this team. | NA | Supported |
+|`TeamsActivity.Send.User`|Send activity notifications to the user. | NA | Supported |
```
Comment (v-sdhakshina) [inline msteams-platform/graph-api/rsc/resource-specific-consent.md:56]:
```suggestion
|`TeamsActivity.Send.User`|Send activity feed notifications to the user. | NA | Supported |
```

**Review by v-sdhakshina — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/31b41de595f00e46bc85af0549a1e888bc3265e8/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 94 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6d1bd736-5c19-4341-8635-4752eece9e61) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-sdhakshina — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bf8a732f8183001da876ff2bf9742cae2af2bd32/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 94 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/53ba0ca2-af7b-4c0e-9131-167c80a020ba) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**